src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball = new Ball();
+        Ball ball1 = new Ball();
+        Ball ball2 = new Ball();
+        Ball[] balls = new Ball[]{ball, ball1, ball2};
+        for (int i = 0; i < balls.length; i++) {
+            System.out.println(balls[i] = lottery.getRandomBall());
```

Comment:
```suggestion
            System.out.println(lottery.getRandomBall());
```
якщо ти зараз зробиш так, результат роботи програми не зміниться. По суті тут краще було б так зробити 
```java
Ball[] balls = new Ball[3]; // пустий масив на 3 комірки 
for (...) {
	balls[i] = lottery.getRandomBall();
	sout(balls[i]);
}
```
Так ти не створюєш пусти об'єкти Ball, які потім замінюєш в циклі а просто створюєш пустий масив з фіксованим розміром